cli/valet.php

```diff
@@ -462,6 +462,28 @@
 
         passthru($command);
     })->descriptions('Tail log file');
+
+    /**
+      * Configure or display the directory-listing setting.
+      */
+    $app->command('directory-listing [status]', function ($status = null) {
+        $key = 'directory-listing';
+        $config = Configuration::read();
+
+        if (is_null($status)) {
```

Comment:
Can we re-write this this way?

```php
        if (in_array($status, ['on', 'off'])) {
            $config[$key] = $status;
            Configuration::write($config);
            return output('Directory listing configuration is saved as '.$status);
        }

        $current = isset($config[$key]) ? $config[$key] : 'off';
        output('Directory listing is '.$current);
```

I know it doesn't give directions but I feel like that's ok. What do you think?

Reply:
Agreed - I like this brevity. Funny how you can stare at something so long and not see the obvious way to simplify! :)